azure/azure_kv.go

```diff
@@ -25,6 +25,8 @@ const (
 	AzureEnviornment = "AZURE_ENVIORNMENT"
 	// AzureVaultURI of azure key vault
 	AzureVaultURL = "AZURE_VAULT_URL"
+	// Default context timeout for Azure SDK API's
+	defaultTimeout = 6000
```

Comment:
nit: Use `6000 * time.Second` as the constant so that you don't have use `XX * time.Second` everywhere.